README.md

```diff
@@ -8,7 +8,7 @@
 - This Github repository explores the website of a Basic Calculator made by Yoshitha Rathnayake.
 - This website shows a preview of my Calculator. 
 - The link for downloading the Calculator app is also available on the website.
-- The programming languages used to build this website are- HTML, CSS and BOOTSTRAP.
+- The programming languages used to build this website are- HTML, CSS and Javascript.
```

Comment:
Don't delete the BOOTSTRAP @Zweak 
```suggestion
- The programming languages used to build this website are- HTML, CSS JAVASCRIPT and BOOTSTRAP.
```

Reply:
> Screenshot needed @Zweak
![Capture5](https://user-images.githubusercontent.com/83528337/198689438-9ba7ddd5-53ef-4596-8f12-224b9087ab78.PNG)

Made another commit, ready for review.